webapp/layouts/README.md

```diff
@@ -0,0 +1,7 @@
+# LAYOUTS
+
+**This directory is not required, you can delete it if you don't want to use it.**
+
+This directory contains your Application Layouts.
+
+More information about the usage of this directory in [the documentation](https://nuxtjs.org/guide/views#layouts).
```

Comment:
Stuff like that could have been hidden with the pre-merge with master. But it's okay.